### PR TITLE
moved image guestbook, added basic image example

### PR DESCRIPTION
**Pull request by thesandlord**
I moved the current contents of "appengine/images" to "appengine/images/guestbook" and added "appengine/images/basic" that will host the code currently on https://cloud.google.com/appengine/docs/python/images/usingimages

### CODE REVIEW

**Comment by jonparrott** [comment]
Can you move /basic to /api, to be consistent with other samples?

File: appengine/images/guestbook/app.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# This file specifies your Python application's runtime configuration
```
Comment (jonparrott) [inline appengine/images/guestbook/app.yaml]: You can strip all of the comments out of this file. They're only needed for hello world.

**Comment by jonparrott** [comment]
LGTM with two minor nits. 👍 Thanks, @thesandlord.

**Comment by thesandlord** [comment]
done and done

**Pull request merged**